src/module.c

```diff
@@ -978,6 +978,27 @@ void RM_ChannelAtPosWithFlags(RedisModuleCtx *ctx, int pos, int flags) {
     res->numkeys++;
 }
 
+/* Returns 1 if name is valid, otherwise returns 0.
+ *
+ * We want to block some chars in module command names that we know can
+ * mess things up.
+ *
+ * There are these characters:
+ * '\t', '\v', '\f', ' ' (space) - issues with old inline protocol.
```

Comment:
where did you find references to `\t`, `\v` and `\f` in the inline protocol?

Reply:
ohh, i did not find these in the inline protocol, i just thought these are spaces, and should not be allowed. should i remove these, or add it to another category?

Reply:
i don't mind blocking them, and i also don't mind not blocking them (sine they won't actually do any damage).
what bothers me more is the fact we associate them with the inline protocol, which is not true.
maybe removing them from the list is simpler than adding another line with another comment.